conf/lookup/BCD-Port.csv

```diff
@@ -11452,7 +11451,7 @@ PortId,CountryCode,PortType,PortCode,KeyWords,Name,Name-cy,Priority,Region,Locat
 31870,UK,2,FIE,,Fair Isle Airport - FIE,Fair Isle Airport - FIE,,1,NULL,1,
 31879,UK,2,FAB,,Farnborough Airport - FAB,Farnborough Airport - FAB,,1,NULL,1,
 31961,UK,2,BHD,,George Best Belfast City Airport - BHD,George Best Belfast City Airport - BHD,5,2,NULL,1,
-31969,UK,2,GLW,,Glasgow International Airport - GLW,Glasgow International Airport - GLW,5,1,NULL,1,
+31969,UK,2,GLW,Bowling,Glasgow International Airport - GLW,Glasgow International Airport - GLW,5,1,NULL,1,
 31971,UK,2,PIK,,Glasgow Prestwick Airport - PIK,Glasgow Prestwick Airport - PIK,5,1,NULL,1,
 31976,UK,3,ULL,,Ullapool Seaport - ULL,Ullapool Seaport - ULL,,1,NULL,1,

```

Comment:
Also in 200043,UK,3,INV,Burghead Corpach Incl Annat Pier Lochaline Lossiemouth Scrabster Ullapool Wick ,Inverness Ports - INV,Inverness Ports - INV,,1,NULL,0,

Just keep both